releasenotes/notes/1.2/backendv1-d0d0642ed38fed3c.yaml

```diff
@@ -1,12 +1,16 @@
 ---
 deprecations_providers:
   - |
-    The :class:`.BackendV1` class is deprecated and it will be removed not earlier than the next major release.
+    The :class:`.BackendV1` class is deprecated and it will be removed no earlier than the next major release.
     There are several migration paths available depending on the main purpose of the backend object::
 
-      - To expose backend information with no access to execution (just a hardware description), consider constructing a :class:`.Target` directly.
-      - To provides access to execution capabilities, consider relying on the primitives interfaces instead.
-      - Alternatively, to continue providing simultaneous :class:`.Target` (hardware information) and ``run`` (execution) capabilities, consider moving to :class:`.BackendV2` (see <https://qisk.it/backendV1-to-V2>).
+      - To expose backend information with no access to execution (just a hardware description), 
+        consider constructing a :class:`.Target` directly.
+      - To provide access to execution capabilities, consider relying on the primitives interfaces 
+        instead.
+      - Alternatively, to continue providing simultaneous :class:`.Target` (hardware information) 
+        and ``run`` (execution) capabilities, consider moving to :class:`.BackendV2` 
+        (see `the migration guide<https://qisk.it/backendV1-to-V2>`_).
```

Comment:
This is a weird suggestion bulleted list, the normal workflow would be to just have people migrate to `BackendV2`. Like I get there is a narrative around using primitives for execution, but migrating to `BackendV2` isn't about execution and a target. You can think of it as providing an interface for building targets. Just saying build a target isn't useful if you're maintaining a provider.